nextjs-sdk/cli.js

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env node
+
+import { exit } from 'process';
+import { exec } from 'child_process';
+import { Octokit } from 'octokit';
+import { existsSync, rmSync, mkdirSync, writeFileSync, readFileSync } from 'fs';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import chalk from 'chalk';
+import ora from 'ora';
+
+const TEMP_DIR_NAME = '.tmp_react_sdk_90723';
```

Comment:
There should be a way to get a temporary folder name assigned by the operating system.